resources/aws/subnet.go

```diff
@@ -100,17 +103,23 @@ func (s *Subnet) CreateOrFail() error {
 }
 
 func (s *Subnet) Delete() error {
```

Comment:
OK, I agree, thanks for clarification. Yes, separating those things later would be good.